tests/jballerina-unit-test/src/test/resources/test-src/jvm/TestProjectNegative/Ballerina.toml

```diff
@@ -0,0 +1,4 @@
+[package]
```

Comment:
This change is not related to JVM stuff. This is a parser related test, Shall we move this project out from jvm package? and move to a place where we keep parser related tests?

Reply:
`parser` module seems good